ci/jenkins/test.sh

```diff
@@ -0,0 +1,464 @@
+    # cache when registry is used.
+    IMAGE_PULL_POLICY="IfNotPresent"
+fi
+
+E2ETEST_PATH=${WORKDIR}/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test
+
+function pull_antrea_ubuntu_image {
+    harbor_images=("base-ubuntu:2.14.0" "golang:1.15")
+    antrea_images=("antrea/base-ubuntu:2.14.0" "golang:1.15")
+    for i in {0..4}; do
+        docker pull ${DOCKER_REGISTRY}/antrea/${harbor_images[i]} && docker tag ${DOCKER_REGISTRY}/antrea/${harbor_images[i]} ${antrea_images[i]} || true
+    done
+}
+
+function export_govc_env_var {
+    export GOVC_URL=$GOVC_URL
+    export GOVC_USERNAME=$GOVC_USERNAME
+    export GOVC_PASSWORD=$GOVC_PASSWORD
+    export GOVC_INSECURE=1
+    export GOVC_DATACENTER=$GOVC_DATACENTER
+    export GOVC_DATASTORE=$GOVC_DATASTORE
+}
+
+function clean_antrea {
+    echo "====== Cleanup Antrea Installation ======"
+    kubectl delete ns antrea-test || true
+    kubectl get pod -n kube-system -l component=antrea-agent --no-headers=true | awk '{print $1}' | while read AGENTNAME; do
+        kubectl exec $AGENTNAME -c antrea-agent -n kube-system -- ovs-vsctl del-port br-int gw0 || true
+    done
+    for antrea_yml in ${WORKDIR}/*.yml; do
+        kubectl delete -f $antrea_yml --ignore-not-found=true || true
+    done
+}
+
+function clean_for_windows_install_cni {
+    # https://github.com/vmware-tanzu/antrea/issues/1577
+    kubectl get nodes -o wide --no-headers=true | awk '$3 != "master" && $1 ~ /win/ {print $6}' | while read IP; do
+        CLEAN_LIST=("/cygdrive/c/opt/cni/bin/antrea.exe" "/cygdrive/c/opt/cni/bin/host-local.exe" "/cygdrive/c/k/antrea/etc/antrea-agent.conf" "/cygdrive/c/etc/cni/net.d/10-antrea.conflist" "/cygdrive/c/k/antrea/bin/antrea-agent.exe")
+        for file in "${CLEAN_LIST[@]}"; do
+            ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "rm ${file}" || true
+        done
+    done
+}
+
+function collect_windows_network_info_and_logs {
+    echo "=== Collecting network info after failure ==="
+    mkdir network_info
+    kubectl get pod -n kube-system -l component=antrea-agent --no-headers=true | awk '{print $1}' | while read AGENTNAME; do
+        mkdir network_info/$AGENTNAME
+        kubectl exec "$AGENTNAME" -c antrea-agent -n kube-system -- ovs-ofctl dump-flows br-int > "network_info/${AGENTNAME}/flows" || true
+        kubectl exec "$AGENTNAME" -c antrea-agent -n kube-system -- ovs-vsctl show > "network_info/$AGENTNAME/db" || true
+        kubectl exec "$AGENTNAME" -c antrea-agent -n kube-system -- ip a > "network_info/${AGENTNAME}/addrs" || true
+    done
+    kubectl get nodes -o wide --no-headers=true | awk '$3 != "master" && $1 ~ /win/ {print $6}' | while read IP; do
+        mkdir network_info/${IP}
+        ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "ovs-ofctl dump-flows br-int" > "network_info/${IP}/flows" || true
+        ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "ovs-vsctl show" > "network_info/${IP}/db" || true
```

Comment:
are these duplicates? we already run these commands in each antrea-agent Pod (line 143-144)

Reply:
Indeed. Removed.